eZ/Publish/API/Repository/ContentService.php

```diff
@@ -432,4 +432,15 @@ public function newTranslationInfo();
      * @return \eZ\Publish\API\Repository\Values\Content\TranslationValues
      */
     public function newTranslationValues();
+
+    /**
+     * Returns true if the given content type $contentType has content instances.
+     *
+     * @since 6.0.1
+     *
+     * @param \eZ\Publish\API\Repository\Values\ContentType\ContentType $contentType
+     *
+     * @return bool
+     */
+    public function isContentTypeUsed(ContentType $contentType);
```

Comment:
@andrerom Looking at the method name it seems to belong to CT service, but since what it actually does is querying Content objects I placed it in Content (i.e. let the content service deal with content, and the CT service deal with CTs)